typescript/src/deposit-sweep.ts

```diff
@@ -130,238 +150,297 @@ export async function assembleDepositSweepTransaction(
     throw new Error("Number of UTXOs must equal the number of deposit elements")
   }
 
-  const walletKeyRing = createKeyRing(walletPrivateKey, witness)
-  const walletAddress = walletKeyRing.getAddress("string")
+  const network = toBitcoinJsLibNetwork(bitcoinNetwork)
+  const keyPair = ecFactory(tinysecp).fromWIF(walletPrivateKey, network)
```

Comment:
`keyPair` -> `walletKeyPair`?

Reply:
Done in 59fb549db1ebf7f92310cd5c1ac05dbdf9f34630.